### ISSUE TITLE
Refine announcing highway refs

### ISSUE THREAD

**Issue by ZeljkoBG** [osmandapp/OsmAnd#19869]
Add an option, trigger, so tts doesn't pronounce the number of road/street. It's just a name.
Most people don't care about the number of roads. And it's irritating.

**Comment by yuriiurshuliak** [comment]
To achieve a more streamlined voiace guidance experience, you can customize the voice prompt settings in the OsmAnd application. By disabling options such as Street names (TTS), Exit number, Traffic warnings, Pedestrian crosswalks, and Tunnels in the voice prompts settings, you can reduce the frequency of voice instructions.
 
To access the voice prompt settings, go to Menu → Navigation → Settings → Sound → Settings.

**Comment by ZeljkoBG** [comment]
What you write has nothing to do with the pronunciation of the street name + "number of road/street".

Some applications, by default, do not read 'number of road/street' and others app have an option to turn it off (trigger).

**Comment by sonora** [comment]
The significance of highway numbers is extremely region-sensitive. In the US, for example, knowing the number of a US, state, or even county road will get you a long way and is very essential. In Germany, the number of motorways (Ax) or national highways (Bx) is significant, too, as found on all signing as well as in traffic announcements. While for lower hierarchy highways (e.g. "Staatsstrassen") the use of the numbers is limited to special cases only, e.g. construction  crew or emergency crew maps.

Long story short, it is not easy to predict in what cases number announcement is essential vs. expendable. But I am certain it is more a regional topic than one of personal preference, e.g. the same user in different regions would in one region need numbers, in another not.

**Comment by ZeljkoBG** [comment]
The solution is to select a trigger. Who needs it should read it. Who should not turn off. Simple.

**Comment by ZeljkoBG** [comment]
Of course it should be an option in the settings for TTS. And where else would you?

The worst option is your suggestion 'region-automated', because no application has that.
'potentially an inferior solution' - the best navigations have a selection (trigger). So 'inferior solution' has been an option for 20 years.
